.changeset/red-impalas-serve.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/techdocs-common': patch
+---
+
+Adds custom docker image support to the techdocs generator. This change allows the `techdocs.generators.techdocs` configuration key to support both an existing string (`docker` or `local`) or an expanded form that adds a new `dockerImage` setting.
+
+```yaml
+techdocs:
```

Comment:
It would be good to update the TechDocs configuration docs page with details (especially useful to include details not just on how, but why folks would want to do this): https://backstage.io/docs/features/techdocs/configuration